### ISSUE TITLE
pertained model

### ISSUE THREAD

**Issue by dogyoonlee** [hyangwinter/flownet3d_pytorch#7]
Can pertained model be applied to kitti dataset?

**Comment by hyangwinter** [comment]
No, the data distributions of these two datasets are very different. If you do this, the result will get a big performance loss. You should retrain this model on KITTI.

**Comment by Haiyan-Chris-Wang** [comment]
@hyangwinter could you explain the reason why the generalization of this model is not good for kitti? In the flownet3d paper, they seem to have a well generalization on the KITTI even without re-train. Is it due to your implementation?

**Comment by hyangwinter** [comment]
Well, I have not tested this model on the KITTI dataset. If you are interested, you can refer to the official implementation to replace FlyingThings3D dataloader with KITTI dataloader.
